### PR TITLE
Change nzbget to full component

### PR DESCRIPTION
**Pull request by chriscla**
**Description:**
Companion pull request to proposed change that will change NZBGet component from solely a sensor platform to a full component

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant/pull/26462

## Checklist:

- [X ] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X ] The documentation follows the [standards][standards].

### CODE REVIEW

**Review by DubhAd — DISMISSED** [review]
Can be merged when parent is merged

File: source/_components/nzbget.markdown
Comment (frenck) [inline source/_components/nzbget.markdown]: I don't think this line is correct though...

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there @chriscla, I've reviewed your PR and thing there is something wrong. Could you please take a look? Thanks! 👍

File: source/_components/nzbget.markdown
Comment (chriscla) [reply source/_components/nzbget.markdown]: I'll fix.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by frenck** [comment]
Thanks for fixing it @chriscla! 👍

**Pull request merged**
